src/request/notification/localized.rs

```diff
@@ -6,29 +6,32 @@ use std::{
     borrow::Cow,
 };
 
-#[derive(Serialize, Debug, Clone)]
+#[derive(Serialize, Deserialize, Default, Debug, Clone)]
 #[serde(rename_all = "kebab-case")]
 pub struct LocalizedAlert<'a> {
-    title: &'a str,
```

Comment:
Instead of having these as public fields, we should utilize the builders as we do with notifications. The API to set optional fields directly is never that great, and will pollute the code with `None` if not needing to set the fields.

Could you add these as builders instead?

Reply:
If you utilize `Default`, it is pretty concise:
```rust
LocalizedAlert {
    title_loc_key: Some("SentTab.NoTabArrivingNotification.title"),
    loc_key: Some("SentTab.NoTabArrivingNotification.body"),
    ..Default::default()
}
```

The whole point of these structs are to map 1:1 to the JSON that APNS expects. It makes sense to at least make the fields public.